src/api/Jalasoft.TeamUp.Projects.API/Controllers/DeleteProjectById.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System;
+    using System.Net;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+
+    public class DeleteProjectById
+    {
+        private readonly IProjectsService projectsService;
+
+        public DeleteProjectById(IProjectsService projectsService)
+        {
+            this.projectsService = projectsService;
+        }
+
+        [FunctionName("DeleteProjectById")]
+        [OpenApiOperation(operationId: "DeleteProjectById", tags: new[] { "Projects" })]
+        [OpenApiParameter(name: "id", In = ParameterLocation.Path, Required = true, Type = typeof(Guid), Description = "The project identifier.")]
+        [OpenApiResponseWithoutBody(statusCode: HttpStatusCode.NoContent, Description = "There's no content to be returned")]
```

Comment:
based on the unit tests a not found scenario needs to be documented as well

Reply:
Done